react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -39,7 +39,8 @@ class InlineDialogFailure extends Component {
             = supportLink
                 ? ( // eslint-disable-line no-extra-parens
                     <div className = 'inline-dialog-error-text'>
-                        <span>{ t('inlineDialogFailure.supportMsg') }</span>
+                        <span>{ `${t('inlineDialogFailure.supportMsg')} `}
```

Comment:
String.prototype.padEnd might be available for use now. Maybe that's worthwhile.